pkgs/servers/matrix-corporal/default.nix

```diff
@@ -0,0 +1,22 @@
+{lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule {
+  pname = "matrix-corporal";
+  version = "1.9.0";
+
+  src = fetchFromGitHub {
+    owner = "devture";
+    repo = "matrix-corporal";
+    rev = "1.9.0";
+    sha256 = "0zwfvlhp9j3skz6ryi01d7ps3wm8kb3njcbjf0bl1gv2h0cjhlji";
+  };
+
+  vendorSha256 = "1fghbl0b418ld5szjafbiz3vp773skc0l6kaif5c2vsh2ivgr6hl";
+
+  meta = with lib; {
+    homepage = "https://github.com/devture/matrix-corporal";
+    description = "Reconciliator and gateway for a managed Matrix server";
+    maintainers = [ teams.matrix.members ];
```

Comment:
* `teams.matrix.members` is already a list, so this should be assigned directly to `maintainers`.
* I'm actually against setting the full team here. Back when I introduced it, I asked a few people about the responsibilities and some were skeptical about maintaining small community-pkgs (like `mautrix-whatsapp` or, well, this one).

Reply:
This is fairly widely used in corporate deployments together with ma1sd, afaik but copy that. Kind of strange to have a "matrix" group that only maintains synapse though :see_no_evil: